qemu/tests/cfg/multi_nics_verify.cfg

```diff
@@ -4,10 +4,14 @@
     no RHEL.3.9
     nics_num = 27
     start_vm = no
+    login_timeout = 1800
     variants:
         - @default:
         - with_multiqueue:
             queues = ${smp}
             vt_ulimit_nofile = 10240
-            Windows..i386:
-                nics_num = 8
+            Windows:
+                smp = 4
+                queues = 4
```

Comment:
@kimi1978wy Would you help explain why set fixed smp and queues for Windows?

Reply:
It is because of windows memory limited